src/components/informationCard/InformationCard.css

```diff
@@ -0,0 +1,32 @@
+.infoCard {
+    flex: 0 1 calc(50% - 30px);
+    background-color: var(--color-background-light);
+    border-radius: 20px;
+    border: none;
+    padding: 30px;
+    position: relative;
+}
+
+.info-card-title {
+    font-size: 35px;
+    font-weight: 400;
+}
+
+.info-card-content {
+    line-height: 2.5;
+}
+
+.info-card-experience-tag {
+    line-height: 2.5;
+}
+
+.arrow-button {
+    position: absolute;
+    right: 15px;
+    bottom: 15px;
+    padding: 15px;
```

Comment:
Padding is top, zo ziet de knop er beter uit, maar de knop is een beetje langwerpig! Kun jij uitzoeken hoe je dat kunt oplossen?

<img width="421" alt="Screenshot 2024-12-05 at 10 52 46" src="https://github.com/user-attachments/assets/dcd9454a-0540-41b3-b77a-2ced83c62163">

Reply:
Ik heb het opgelost door van de button een flexbox te maken met een justify-content property: center.